src/Math/Matrix4.elm

```diff
@@ -45,6 +45,11 @@ transform = Native.MJS.v3mul4x4
 identity : Mat4
 identity = Native.MJS.m4x4identity
 
+{-| Computes the inverse of the given matrix m.
+-}
+inverse : Mat4 -> Mat4
```

Comment:
It'd be nice if this could be `Mat4 -> Maybe Mat4` instead, but I don't know how to achieve that.